classes_functions/recursive.py

```diff
@@ -0,0 +1,9 @@
+# Source: https://www.w3schools.com/python/python_functions.asp
+
+def tri_recursion(k):
```

Comment:
Why "tri" in the name?

Reply:
the function is straight copied from w3schools

Reply:
I suppose I can't question it too much then, oh well.